### PR TITLE
[Api][Administrator] refactor endpoints for admin and admin avatar

### PR DESCRIPTION
**Pull request by AdamKasp**
| Q               | A
| --------------- | -----
| Branch?         | 1.8
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Related tickets | #11250 
| License         | MIT

### CODE REVIEW

File: src/Sylius/Behat/Client/Request.php [hunk 1]
```diff
@@ -211,4 +211,13 @@ private function mergeArraysUniquely(array $firstArray, array $secondArray): arr
 
         return $firstArray;
     }
+
+    private static function prepareSection(?string $section): string
```
Comment (AdamKasp) [inline src/Sylius/Behat/Client/Request.php:215]: this function should be deleted after refactor all new API endpoints, for now only refactored endpoint have `section` prefix, without this logic a have problems with pathnames while testing.

Comment (lchrusciel) [reply src/Sylius/Behat/Client/Request.php:215]: IMHO, we can leave it for the future, to give possibility of using this class without any section

File: src/Sylius/Behat/Client/ApiPlatformClient.php [hunk 2]
```diff
@@ -30,19 +30,27 @@ final class ApiPlatformClient implements ApiClientInterface
...
     /** @var RequestInterface */
     private $request;
 
-    public function __construct(AbstractBrowser $client, SharedStorageInterface $sharedStorage, string $resource)
-    {
+    public function __construct(
+        AbstractBrowser $client,
+        SharedStorageInterface $sharedStorage,
+        string $resource,
+        string $section = null
```
Comment (lchrusciel) [inline src/Sylius/Behat/Client/ApiPlatformClient.php:43]:
```suggestion
        ?string $section = null
```

**Review by lchrusciel — APPROVED** [review]

File: src/Sylius/Behat/Client/Request.php (hunk 1 above)
Comment (AdamKasp) [reply src/Sylius/Behat/Client/Request.php:215]: I will fix it in next PR

File: src/Sylius/Behat/Client/ApiPlatformClient.php (hunk 2 above)
Comment (AdamKasp) [reply src/Sylius/Behat/Client/ApiPlatformClient.php:43]: I will fix it in the next PR.

**Review by lchrusciel — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thank you, Adam! :1st_place_medal:

**Review by Tomanhez — APPROVED** [review]

**Pull request merged**